Makefile

```diff
@@ -12,7 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# ARCH is still supported, but prefer BUILDARCH and TARGETARCH.
 ARCH ?= amd64
+BUILDARCH ?= amd64
+TARGETARCH ?= $(ARCH)
```

Comment:
What's the difference between BUILDARCH and TARGETARCH? I'm guessing BUILDARCH refers to the machine performing the build, and the TARGETARCH is the architecture the binary is built for?

Reply:
Right. Naming slightly influenced by https://docs.docker.com/build/building/multi-platform/#cross-compilation

Reply:
BUILDARCH and TARGETARCH are user-facing, how about we add a comment to tell the difference?

Reply:
> how about we add a comment to tell the difference?

Done.